dev/builddeps-veloxbe.sh

```diff
@@ -208,6 +208,8 @@ fi
 concat_velox_param
 
 function build_arrow {
+  get_velox
+  setup_dependencies
```

Comment:
If we add them here, they will be executed two times in the following complete build at line 306.
```
get_velox
setup_dependencies
build_velox_backend
```

Can we move `get_velox` into `setup_dependencies` and explicitly add `setup_dependencies` in dockerfile?

`./dev/builddeps-veloxbe.sh --run_setup_script=ON setup_dependencies build_arrow`

Reply:
let me add a flag to check if we need to run get_velox